packages/react-dev-utils/ModuleScopePlugin.js

```diff
@@ -13,8 +13,9 @@ const chalk = require('chalk');
 const path = require('path');
 
 class ModuleScopePlugin {
-  constructor(appSrc) {
+  constructor(appSrc, allowedPaths) {
     this.appSrc = appSrc;
+    this.allowedPaths = new Set(allowedPaths);
```

Comment:
I assumed this is safe when `allowedPaths` is undefined? We could add a default above to be explicit:
```js
constructor(appSrc, allowedPaths = []) {
```